comp/core/agenttelemetry/impl/config.go

```diff
@@ -257,6 +257,10 @@ var defaultProfiles = `
         - name: logs.destination_http_resp
           aggregate_tags:
             - status_code
+        - name: oracle.activity_samples_count
+        - name: oracle.activity_latency
+        - name: oracle.statement_metrics
+        - name: oracle.statement_plan_errors
```

Comment:
@iglendd is this something we want to add to the agent telemetry?

Reply:
Yes. we do we. This is what telemetry is for. Obviously it will be operational only of the integration is enabled.

Reply:
> Obviously it will be operational only of the integration is enabled.

Do you mean if the agenttelemetry check is enabled or if the Oracle integration is enabled? It seems odd to me that we're adding oracle metrics here. Shouldn't they just be part of the oracle check instead?

Reply:
It is a good question. Currently, and I think in future, for a number reasons, including non-technical, Agent Telemetry metrics need to be explicitly stated in the specified configuration in order to be emitted. If they are not provided at the source, which would happen naturally if the integration is not running Oracle (that is what I have actually tried to say in the abbreviated manner), it will not exist or be found by Agent Telemetry component and hence will not be emitted. It does constitute a form of, usually undesired coupling, but first that kind of coupling is too weak to bother with IMHO, and second, we need to have a central, explicit and formal control of what actually would be emitted, to avoid certain phenomena, e.g. unintended data leakage.

Reply:
Another thing is transparency for customers who may want to understand what exactly will be sent.

Reply:
Does that mean eventually we will have more integration telemetry metrics inside the agent? eg: db integration from this list:
https://docs.datadoghq.com/database_monitoring/